infra/helper.py

```diff
@@ -215,7 +215,7 @@ def _build_image(image_name, no_cache=False, pull=False):
 
 def docker_run(run_args, print_output=True):
   """Call `docker run`."""
-  command = ['docker', 'run', '--rm', '-i', '--privileged']
+  command = ['docker', 'run', '--network=host', '--rm', '-i', '--privileged']
```

Comment:
is there a difference between `--net` and `--network`? Did you intentionally upload changes for this file, or it's just your local copy? If it's something we should improve, could you please upload it separately from the project integration and add some clarification about that.

Reply:
Woops, I did not intend to commit that here, indeed (I just came back to this to try to understand what was wrong). 

`--net` and `--network` are afaik the same thing (I'm still not sure, I think --net is the "old" option, and was deprecated in favour of --network )
One of the computers I work on is in a network with bridges and a custom DNS server (in an academic network), so the default behavior of docker does not have internet access (or just DNS? anyway, it's not very practical) and this tells docker to directly use the host's connection (which works). I think I forgot to modify the "docker pull" request at line 254 which I had also edited at some point.

I have no idea if adding --network=host by default has drawbacks (probably, or it would be the default, I guess), but if there could be an argument to helper.py to pass the option to docker, it'd be nice (I can try it in another PR)